### ISSUE TITLE
Allow for inline comments that are ignored by the calculator

### ISSUE THREAD

**Issue by meglio** [ovk/clcalc#115]
When creating an explanatory step-by-step calculation that I then want to share with someone, I would like to enter comments and have them greyed out and ignored.

**Comment by ovk** [comment]
You can use `#` character to create a comment, here is an [example](https://clcalc.net#eyJjIjpbIiPCoFRoaXPCoGlzwqBleGFtcGxlwqBjb21tZW50IiwiMsKgK8KgMiIsIjTCoCrCoDLCoCPCoFRoaXPCoGlzwqBpbmxpbmXCoGNvbW1lbnQiXSwidiI6IjEifQ==).

Is this what you were looking for?

**Comment by meglio** [comment]
Yes, exactly, thanks. It's worth adding to the quickstart guide - it is especially useful for sharing. Also, a different color highlight for comments would be useful.

Also, deleting blocks would be helpful.

**Comment by ovk** [comment]
Good point about documentation. And thanks for suggestion for deleting blocks.

As for highlighting comments, there are few issues I want to fix with syntax highlighting in general, and those would cover comments as well.
